src/global.F90

```diff
@@ -104,6 +104,19 @@ module global
   ! What to assume for expanding natural elements
   integer :: default_expand = ENDF_BVII1
 
+  ! Random number seed for cross sections, specially for URR ptables
+  ! This number is shared by all nuclides and updated after particle
+  ! changed its energy.
+  integer(8) :: xs_seed = 1_8
```

Comment:
Rather than adding a separate global variable for `xs_seed`, I would prefer that we create a new stream.  Perhaps `STREAM_URR_PTABLE`